pkg/test/istio.io/examples/testType.go

```diff
@@ -31,19 +31,28 @@ type testStep interface {
 type fileTestType struct {
 	path      string
 	namespace string
+	delete    bool
 }
 
-func newStepFile(namespace string, path string) testStep {
+func newStepFile(namespace string, path string, delete bool) testStep {
```

Comment:
Does it make sense to have a delete? Theoretically the cleanup scripts at the end of each test should clean these up. 

I wonder if we could run through a cleanup process at the end of .Run that would attempt to clean up all of the resources and error if any still exist. Would have to support both yamls and anything in the scripts though.

Reply:
Delete is not only for cleanup, some of the tasks asks explicitly to delete some policy to see the new effect, it's part of the task.

Reply:
Also moved the real clean to the script to make this more clear.